news/_posts/2021-04-28-new-repository-service.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: New repository service for spark-packages
+categories:
+- News
+tags: []
+status: publish
+type: post
+published: true
+meta:
+  _edit_last: '4'
+  _wpas_done_all: '1'
+---
+We have spun up a new repository service at <a href="https://repos.spark-packages.org">https://repos.spark-packages.org</a> and it will be the new home for the artifacts on spark-packages.
+
+<a href="https://bintray.com/">Bintray</a>, the original repository service used for <a href="https://spark-packages.org/">https://spark-packages.org/</a>, is in its <a href="https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/">sunset process</a>. It will no longer be available from May 1st. To consume artifacts from that, please replace "dl.bintray.com/spark-packages/maven" with "repos.spark-packages.org" in the Maven pom files or sbt build files in your repositories.
```

Comment:
Is the setting in pom file or sbt build file? I thought it's controlled by the system env `DEFAULT_ARTIFACT_REPOSITORY`. See https://github.com/apache/spark/commit/f738fe07b6fc85c880b64a1cc2f6c7cc1cc1379b#diff-f8564df81d845c0cd2f621bc2ed22761cbf9731f28cb2828d9cbd0491f4e7584R1201

Reply:
The system env is specific usage for Spark. The end-users should just care about their pom/build file.

Reply:
For spark release, we need to set this env but I think after we backport the change in all release branches, it should not be a problem.